index.html

```diff
@@ -355,7 +357,8 @@ <h3 class="footer-header">Contact me</h3>
               ></textarea>
             </li>
           </ul>
-          <button type="submit" class="btn5">Get in touch</button>
+          <p id="invalid"></p>
+          <button id='press' type="submit" class="btn5">Get in touch</button>
```

Comment:
- [ ] Try to use a brighter colour (one that fits your design template) for the error message

Yours | My try
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124259224-2fac6700-db26-11eb-98c0-075bf3aa5759.png) | ![image](https://user-images.githubusercontent.com/16058470/124259172-23c0a500-db26-11eb-9b99-0b334f46847a.png)

Reply:
Done